README.md

```diff
@@ -54,10 +54,30 @@ Flags:
                                 Metrics Namespace ($METRICS_NAMESPACE)
   --metrics_path="/metrics"     Path under which to expose metrics for the bridge ($METRICS_PATH)
   --extended_details            When enabled, alerts are presented in HTML format and include colorized status (FIR|RES), alert start time, and a link to the generator of the alert ($EXTENDED_DETAILS)
+  --dispatch_errors             When enabled, alerts will be tried to dispatch with a error-message regarding faulty templating or missing fields to help debugging ($DISPATCH_ERRORS)
   --debug                       Enable debug output of the server
   --version                     Show application version.
 ```
 
+### Templating
+The bridge now supports [Go templating](https://golang.org/pkg/text/template/), so you can customize the alertmessages further.  
```

Comment:
Can we extend this sentence just a little bit to make it clear where the customization needs to happen? Maybe...
> ... so you can customize the alert messages further with templates in the title and message annotations

I request this because I was wondering at first look where the template text came from and realized that one must configure alertmanager to send the templated fields.

Reply:
Yes of course. I hope this is now clearer.